examples/tutorials/31_Introduction_To_Material_Science.ipynb

```diff
@@ -0,0 +1,428 @@
+   "execution_count": 14,
+   "id": "d6c18dd9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset_config = {\"reload\": True, \"featurizer\": dc.feat.CGCNNFeaturizer(), \"transformers\": []}\n",
+    "tasks, datasets, transformers = dc.molnet.load_perovskite(**dataset_config)\n",
+    "train_dataset, valid_dataset, test_dataset = datasets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "129229ed",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<bound method DiskDataset.get_data_shape of <DiskDataset X.shape: (15142,), y.shape: (15142,), w.shape: (15142, 1), task_names: ['formation_energy']>>"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "train_dataset.get_data_shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d36a9822",
+   "metadata": {},
+   "source": [
+    "## Predicting Formation Energy  <a class=\"anchor\" id=\"pred-props\"></a>\n",
+    "\n",
+    "Along with the dataset and featurizers, DeepChem also provide implementation of various machine learning algorithms which can be used on the fly for material science applications. For predicting formation energy, we use `CGCNNModel` as described in the paper [1]."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "4217a819",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.026785855633871897"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model = dc.models.CGCNNModel(mode='regression', batch_size=32, learning_rate=0.001)\n",
+    "model.fit(train_dataset, nb_epoch=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3a41acc6",
+   "metadata": {},
+   "source": [
+    "Once fitting the model, we evaluate the performance of the model using mean squared error metric since it is a regression task. For selection a metric, `dc.metrics.mean_squared_error` function can be used and we evaluate the model by calling `dc.model.evaluate`.`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "57f3a654",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Training set score: {'mean_squared_error': 0.017375291246604516}\n",
+      "Test set score: {'mean_squared_error': 0.01756353898822416}\n"
+     ]
+    }
+   ],
+   "source": [
+    "metric = dc.metrics.Metric(dc.metrics.mean_squared_error)\n",
+    "print(\"Training set score:\", model.evaluate(train_dataset, [metric], transformers))\n",
+    "print(\"Test set score:\", model.evaluate(test_dataset, [metric], transformers))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df2e9207",
+   "metadata": {},
+   "source": [
+    "# Congratulations! Time to join the Community!\n",
```

Comment:
If you want to add links for further reading here, feel free to grab some from my list or blog post, or link to them directly: https://github.com/ncfrey/resources
https://towardsdatascience.com/getting-started-in-materials-informatics-41ee34d5ccfe